contributors/devel/release/issues.md

```diff
@@ -0,0 +1,195 @@
+- *code freeze*: starts 2 weeks after code slush.  only critical bug fixes are accepted into the release codebase.
+
+## Requirements for adding an issue to the milestone
+
+**Note**: Issues with unmet label requirements will automatically be removed from the release milestone.
+
+When adding an issue to a milestone, the Kubernetes bot will check that the following
+labels are set, and comment on the issue with the appropriate instructions.  The
+bot will attempt to contact the issue creator 3 times (over 3 days)
+before automatically removing the issue from the milestone.
+
+Label categories:
+
+- SIG label owner
+- Priority
+- Severity
+- Issue type
+
+### SIG owner label
+
+The SIG owner label defines the SIG to which the bot will escalate if the issue is not resolved
+or updated by the deadline.  If there are no updates after escalation, the
+issue may automatically removed from the milestone.
+
+e.g. `sig/node`, `sig/federation`, `sig/apps`, `sig/network`
+
+### Priority
+
+Priority label used by the bot to determine escalation path before moving an issues
+out of the release milestone.
+
+- `priority/critical-urgent`: Never automatically move out of a release milestone; continually escalate to contributor and SIG through all available channels.- `priority/critical-urgent`: Never automatically move out of a release milestone; continually escalate to contributor and SIG through all available channels.
+- `priority/important-soon`: Escalate to the issue owners and SIG owner; move out of milestone after several unsuccessful escalation attempts.
+- `priority/important-longterm`: Escalate to the issue owners; move out of the milestone after 1 attempt.
+
+### Severity
+
+The severity is used to triage which issues the release team needs to track as part of cutting
+a release.  As the release date approaches, release blocking issues are expected to be updated
+frequently and have an issue owner present at all burn down meetings until the issue is resolved.
+
+- `release/blocker`: Will hold the release until this is resolved.
+  - post code slush: issue owner update frequency: every 3 days
+  - code freeze: issue owner update frequency: daily
+  - would require a patch release if left undiscovered until after the minor release.
+  - **must** have critical-urgent priority.
+- `release/non-blocker`: Will not hold the release if this issue is unresolved on the target release date.
+  - would not require a patch release
+  - will automatically be moved out of the release milestone at code freeze
+  - **must not** have critical-urgent priority.
+
+### Issue type
+
+The issue type is used to help identify the types of changes going into the release over time.
+This will allow us to develop a better understanding of what sorts of issues we would miss
+with a faster release cadence.
+
+This will also be used to escalate to the correct SIG GitHub team.
+
+- `kind/bug`: Fixes a bug.
+  - were not known at the time the feature was implemented.
```

Comment:
Oh! I wasn't asking if kind/bug was required. I totally accept that. I was asking if the text "were not known at the time the feature was implemented." was necessary.